meson.build

```diff
@@ -43,6 +43,7 @@ features = {
     'ffmpeg': true,
     'gpl': get_option('gpl'),
     'jpegxl': libavformat.version().version_compare('>= 59.27.100'),
+    'av1': libavformat.version().version_compare('>= 59.24.100'),
```

Comment:
Calling it `av1` is a bit misleading, I think avif works better. Same for waf

Reply:
I guess you're right.

Reply:
On another note, that'd mean that `HAVE_AV1` is renamed to `HAVE_AVIF`, which may lead to someone thinking that we have libavif (which we don't), and not just a muxer in libavformat.

Reply:
I'd personally call it `AVIF_MUXER` as is done in ffmpeg. It has a nice ring to it's name, doesn't it?